assets/tests/components/detours/detourMap.test.tsx

```diff
@@ -10,6 +10,10 @@ beforeEach(() => {
   jest.spyOn(global, "scrollTo").mockImplementationOnce(jest.fn())
 })
 
+jest.mock("../../../src/api", () => ({
+  fetchDetourDirections: jest.fn(() => Promise.resolve(null)),
+}))
+
```

Comment:
suggestion(non-blocking): just wondering, did you try this to see if it worked?

```suggestion
jest.mock("../../../src/api")
```

We're intending to [⚙️ Ensure frontend test isolation by resetting mocks](https://app.asana.com/0/1148853526253420/1203482629607208/f) and this default implementation would be lost when resetting the mock.

Instead, you'd want to mock the function in a `beforeEach` call so that it's mocked for every test individually. And then we can use the `mockResolvedValue` helper for this instance.

```suggestion
jest.mock("../../../src/api")

beforeEach(() => {
	jest.mocked(fetchDetourDirections).mockResolvedValue(null)
})
```